src/modules/ModuleLiquidity/View.vue

```diff
@@ -25,7 +25,7 @@ const { loading: isLoading, result, refetch } = useLiquidityPairsQuery(pollInter
 // Refetch if cached
 if (result.value && !isLoading.value) refetch()
 
-const isLoaded = computed(() => !!result.value)
+// const isLoaded = computed(() => !!result.value)
```

Comment:
Do we need it anymore? I think we can use isLoading to display loader instead of "Empty"

Reply:
Yeah, I think we can. But I intentionally avoid to do it, because I think it could be solved in the whole app in general in a single run. There are many places where loaders should be nice.